testcafe/editResource.js

```diff
@@ -11,7 +11,6 @@ fixture`Edit Resource`
 
 
 async function testEditTextProperty(t, showPageSelector, editPageSelector, newValue) {
-  await resourcePage.clickEditButton(t);
```

Comment:
If we're getting rid of the edit button, then you'll still need to navigate to the edit page if you want these tests to pass. You'll want to use the [`navigateTo()`](https://devexpress.github.io/testcafe/documentation/test-api/actions/navigate.html) action.

Reply:
Thanks for the tip! That makes sense. I tried this and I'm getting Test Cafe DNS (cannot resolve url) errors locally. I googled the error and it looks like an issue other people run into with test cafe on localhost so I'm pushing up the changes to see if Travis runs into it too. If it does, have you seen this error before?